### PR TITLE
New working clones

### PR DESCRIPTION
**Pull request by clawgrip**
------------------
Reality Tennis (set 2) [Maria Morpalier, jordigahan, ClawGrip]

Also fixed the clock per the xtal on the PCB

### CODE REVIEW

File: src/mame/drivers/rltennis.cpp [hunk 1]
```diff
@@ -74,11 +74,11 @@ player - when there's nothing to play - first, empty 2k of ROMs are selected.
 
 #define RLT_REFRESH_RATE   60
 #define RLT_TIMER_FREQ     (RLT_REFRESH_RATE*256)
-#define RLT_XTAL           XTAL(12'000'000)
+#define RLT_XTAL           XTAL(32'000'000)
 
 uint16_t rltennis_state::io_r()
 {
-	return (ioport("P1" )->read()&0x1fff) | (m_unk_counter<<13); /* top 3 bits controls smaple address update */
+	return (ioport("P1" )->read()&0x1fff) | (m_unk_counter<<13); // Top 3 bits controls smaple address update
```
Comment (cuavas) [inline src/mame/drivers/rltennis.cpp:81]: If you're reformatting the comment, you should probably fix the spelling error and grammar error while you’re causing diff noise on the line.

File: src/mame/drivers/rltennis.cpp [hunk 2]
```diff
@@ -183,7 +183,7 @@ void rltennis_state::ramdac_map(address_map &map)
 
 void rltennis_state::rltennis(machine_config &config)
 {
-	M68000(config, m_maincpu, RLT_XTAL/2); /* 68000P8  ??? */
+	M68000(config, m_maincpu, RLT_XTAL/8); // MC68000P8
```
Comment (cuavas) [inline src/mame/drivers/rltennis.cpp:186]: Did you confirm the divider as well?

File: src/mame/drivers/rltennis.cpp (hunk 1 above)
Comment (clawgrip) [reply src/mame/drivers/rltennis.cpp]: Fixed, thanks!

Comment (cuavas) [reply src/mame/drivers/rltennis.cpp]: Not quite – remember English words for plurals: "He controls the mafia," vs "They control the mafia."

Comment (clawgrip) [reply src/mame/drivers/rltennis.cpp]: Thanks again!

**Pull request merged**